src/main/java/ch/njol/skript/Skript.java

```diff
@@ -489,6 +489,10 @@ public void onEnable() {
 			classLoadError = e;
 		}
 
+		// Disable pausing
+		if (Skript.methodExists(Server.class, "allowPausing", Plugin.class, Boolean.class))
+			getServer().allowPausing(this, false);
```

Comment:
Do you think there should be a warning message Skript.warning() notifying the server owner that Skript has disabled server pausing?